cypress/integration/tests/developer/applicationinventory/assessment/create.test.ts

```diff
@@ -150,4 +155,26 @@ describe("Application validations", { tags: "@tier2" }, () => {
         cy.wait("@getApplication");
         notExists(application.name);
     });
+
+    it("Bulk deletion of applications", function () {
+        // Click dropdown toggle button to make 'Select page' selection.
+        cy.get("button[aria-label='Select']").click();
+        cy.get("ul[role=menu] > li").contains("Select all").click();
+
+        // Select all the applications on the current page and delete the applications
+        cy.get("ul[role=menu] > li").contains("Select page").click();
+        cy.get(actionButton).eq(1).click();
+        cy.get("a.pf-c-dropdown__menu-item").contains("Delete").click();
+        clickByText(button, "Delete");
```

Comment:
Isn't there a method to click on kebab menu and select the one of the option like "import" , "Delete" etc .. If not let's create this method in applications.ts as this will common for both assessment and analysis page.
So line 166 to 168 can be part of method and you call it here .

Reply:
As of now, there are multiple functions in utils.ts to make kebab menu selections - Import, Manage imports etc. The multiple functions would have to be replaced with a single function and all the tests that use the function(s) would have to be run . 
I'll do the refactoring in a follow up PR .